### ISSUE TITLE
store_model_attributes is causing rails to fail

### ISSUE THREAD

**Issue by neohunter** [grosser/gettext_i18n_rails#160]
After run store_model_attributes tasks, rails server started to complain about this:

`ActionView::Template::Error: can not load translations from config/locales/model_attributes.rb: expects it to return a hash, but does not`

I fixed it by placing `{}` at the end of the file.

**Comment by grosser** [comment]
yeah ... it should not be loaded by rails ... needs to be in a different location :/

can you make a PR to either:
 - place {} at the end
 - move it to a new location

**Comment by neohunter** [comment]
Done,

https://github.com/grosser/gettext_i18n_rails/pull/170
